### PR TITLE
Update 2018-08-24-running-strimzi-on-aks.md

### PR DESCRIPTION
**Pull request by tombentley**
Some suggested changes, trying to make speak more directly to the reader. Wdyt?

### CODE REVIEW

File: _posts/2018-08-24-running-strimzi-on-aks.md [hunk 1]
```diff
@@ -1,31 +1,31 @@
 ---
 layout: post
-title:  "Running Strimzi on Azure Container Service"
+title:  "Get Kafka on Azure Container Service in 15 minutes using Strimzi"
```
Comment (ppatierno) [inline _posts/2018-08-24-running-strimzi-on-aks.md]: "Get Apache Kafka ..." ?

File: _posts/2018-08-24-running-strimzi-on-aks.md [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
-Using the Azure portal through a web browser or the Azure CLI (Command Line Interface), it's just a matter of a few clicks and/or commands and a few minutes to have a fully functional Kubernetes cluster that we can install out containarized applications on top of.
-This is really a great way to have Strimzi up and running in the cloud and then deploying an Apache Kafka cluster through it.
+Using the Azure portal through a web browser or the Azure CLI (Command Line Interface), it's just a matter of a few clicks and/or commands and a few minutes to have a fully functional Kubernetes cluster where we can install our containarized applications.
+This is a really great way to have Strimzi up and running in the cloud and then deploying an Apache Kafka cluster through it.
 
-We're going to show you how easy it is to deploy Strimzi on AKS.
+So in this post I'm going to show you how easy it is to deploy Strimzi on AKS using the CLI.
 If you already have an Azure account you can be up and running with Strimzi in 15 minutes.
+If not you can sign up [here](TODO).
```
Comment (ppatierno) [inline _posts/2018-08-24-running-strimzi-on-aks.md]: the TODO = https://azure.microsoft.com

File: _posts/2018-08-24-running-strimzi-on-aks.md (hunk 1 above)
Comment (tombentley) [reply _posts/2018-08-24-running-strimzi-on-aks.md]: Actually how about: "Apache Kafka in the Cloud in 15 minutes using Strimzi and AKS".

Comment (ppatierno) [reply _posts/2018-08-24-running-strimzi-on-aks.md]: Better! Go for it!

**Pull request merged**